styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -49,7 +55,9 @@ import templateImportant from './_important.html.haml';
   </Story>
 </Preview>
 
-## Adviser
+## Adviser callout
+
+Content that is for advisers. This will only show on the logged-in public site and AdviserNet.
```

Comment:
"This will only show on the logged-in public site and AdviserNet". Perhaps we could reframe this to talk about intent (i.e. if another product is looking to use this component).

Something like: "Content that is for advisers. These should not be shown to the public, and you should only show this when an adviser is logged in"

Reply:
Sounds good, will update. Thanks!